src/04-modality-specific-files/05-task-events.md

```diff
@@ -33,16 +33,15 @@ Each task events file REQUIRES a corresponding task imaging data file
+|-----------------|-----------------------|---------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| onset           | REQUIRED              | [number][]    | Onset (in seconds) of the event measured from the beginning of the acquisition of the first volume in the corresponding task imaging data file. If any acquired scans have been discarded before forming the imaging data file, ensure that a time of 0 corresponds to the first image stored. In other words negative numbers in "onset" are allowed<sup>5</sup>. |
+| duration        | REQUIRED              | [number][]    | Duration of the event (measured from onset) in seconds. Must always be either zero or positive. A "duration" value of zero implies that the delta function or event is so short as to be effectively modeled as an impulse.                                                                                                                                        |
+| sample          | OPTIONAL              | [number][]    | Onset of the event according to the sampling scheme of the recorded modality (that is, referring to the raw data file that the `events.tsv` file accompanies).                                                                                                                                                                                                     |
+| trial_type      | OPTIONAL              | [string][]    | Primary categorisation of each trial to identify them as instances of the experimental conditions. For example: for a response inhibition task, it could take on values "go" and "no-go" to refer to response initiation and response inhibition experimental conditions.                                                                                          |
+| response_time   | OPTIONAL              | [number][]    | Response time measured in seconds. A negative response time can be used to represent preemptive responses and "n/a" denotes a missed response.                                                                                                                                                                                                                     |
+| value           | OPTIONAL              | ????          | Marker value associated with the event (for example, the value of a TTL trigger that was recorded at the onset of the event).                                                                                                                                                                                                                                      |
```

Comment:
Not sure what Datatype this is.

Reply:
I have seen strings and numbers. For example in BrainVision files you can find a marker called `S  1` (string) that gets written when a TTL byte of `1` is received by the recording hardware. Then again some other hardware may simply encode that as `1` (number).

We could also keep it unspecified, or put something like `string, or number, or any other datatype`, I am not sure at all.

Reply:
looking at this again, let's say 
```
[string][] or [number][]
```